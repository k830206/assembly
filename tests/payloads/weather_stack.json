{
  "request": {
    "type": "City",
    "query": "Melbourne, Australia",
    "language": "en",
    "unit": "m"
  },
  "location": {
    "name": "Melbourne",
    "country": "Australia",
    "region": "Victoria",
    "lat": "-37.817",
    "lon": "144.967",
    "timezone_id": "Australia\/Melbourne",
    "localtime": "2020-02-21 00:42",
    "localtime_epoch": 1582245720,
    "utc_offset": "11.0"
  },
  "current": {
    "observation_time": "01:42 PM",
    "temperature": 14,
    "weather_code": 302,
    "weather_icons": [
      "https:\/\/assets.weatherstack.com\/images\/wsymbols01_png_64\/wsymbol_0034_cloudy_with_heavy_rain_night.png"
    ],
    "weather_descriptions": [
      "Rain Shower"
    ],
    "wind_speed": 13,
    "wind_degree": 220,
    "wind_dir": "SW",
    "pressure": 1018,
    "precip": 0.1,
    "humidity": 82,
    "cloudcover": 75,
    "feelslike": 13,
    "uv_index": 1,
    "visibility": 10,
    "is_day": "no"
  }
}