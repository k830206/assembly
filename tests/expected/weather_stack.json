{
  "wind_speed": 13,
  "temperature_degrees": 14
}